doc/conf.py

```diff
@@ -240,6 +242,37 @@
 
 numpydoc_show_class_members = False
 
+# Make numpydoc to generate plots for example sections
+numpydoc_use_plots = True
+
+plot_pre_code = """
+import networkx as nx
+import numpy as np
+np.random.seed(7)
+"""
+
+plot_formats = [("png", 100), "pdf"]
+
+import math
+
+phi = (math.sqrt(5) + 1) / 2
+
+plot_rcparams = {
+    "font.size": 8,
+    "axes.titlesize": 8,
+    "axes.labelsize": 8,
+    "xtick.labelsize": 8,
+    "ytick.labelsize": 8,
+    "legend.fontsize": 8,
+    "figure.figsize": (3 * phi, 3),
+    "figure.subplot.bottom": 0.2,
+    "figure.subplot.left": 0.2,
+    "figure.subplot.right": 0.9,
+    "figure.subplot.top": 0.85,
+    "figure.subplot.wspace": 0.4,
+    "text.usetex": False,
+}
+
```

Comment:
I think we could safely remove all of this for now. I just built the docs locally without these options and the image actually looked better IMO. We can always revisit to tweak the rcparams after we've added more examples!

```suggestion
```

Reply:
Totally!